kwok/cloudprovider/cloudprovider.go

```diff
@@ -211,6 +211,11 @@ func addInstanceLabels(labels map[string]string, instanceType *cloudprovider.Ins
 			ret[r.Key] = r.Values()[0]
 		}
 	}
+	for _, r := range offering.Requirements {
```

Comment:
I think this bit means we can drop some of the static things that we are setting around zone and capacity type below